.evergreen/generate_evergreen_tasks.js

```diff
@@ -120,7 +120,7 @@ BASE_TASKS.push({
   commands: [
     updateExpansions({
       VERSION: 'latest',
-      TOPOLOGY: 'sharded_cluster',
+      TOPOLOGY: 'server',
```

Comment:
There's no real reason we need to run these tests against a sharded cluster with 2 mongoses. We can only do a `w: majority` on user management commands so we were hitting an edge case where the user hadn't completely been dropped on all nodes. before the next test ran. Using a standalone now fixes this issue.